packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -591,6 +599,52 @@ describe('Validation at REST level', () => {
     });
   });
 
+  context(
+    'for request body specified via model definition with strict false',
+    () => {
+      @model({settings: {strict: false}})
+      class ProductNotStrict {
+        @property()
+        id: number;
+
+        @property({required: true})
+        name: string;
+
+        constructor(data: Partial<ProductNotStrict>) {
+          Object.assign(this, data);
+        }
+      }
+
+      class ProductController {
+        @post('products')
+        async create(
+          @requestBody({required: true}) data: ProductNotStrict,
+        ): Promise<ProductNotStrict> {
+          return new ProductNotStrict({id: 1, name: 'pencil'});
```

Comment:
Can you please add a code comment to explain why you are using hard-coded model properties instead of passing the `data` argument? I assume this is needed to bypass validation at juggler level and ensure we reject the request at REST API layer.

Reply:
@ShaunSpinelli same question as @bajtos mentioned, any reason why not use:

```ts
return new Product(data);
```
?

Reply:
Hey @bajtos and @jannyHou. Thanks for the feedback,  that was a mistake and it has been corrected to: 

```javascript
return new ProductNotStrict(data);
```